public/main.js

```diff
@@ -67,3 +69,107 @@ artistName.addEventListener('keypress', function (e) {
       getArtistName();
   }
 });
+
+
+var list = document.getElementById('list');
+var showID = list.addEventListener('click', myClick, false);
+
+var showAlTracks = list.addEventListener('click', userClick, false);
+
+function myClick() {
```

Comment:
Try to make a more meaningful name for `myClick`. The name should have something to do with the action it performs.